driver-sync/src/test/functional/com/mongodb/client/unified/UnifiedTestSkips.java

```diff
@@ -17,26 +17,275 @@
 package com.mongodb.client.unified;
 
 import com.mongodb.assertions.Assertions;
+import com.mongodb.lang.NonNull;
+import com.mongodb.lang.Nullable;
 
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.function.Supplier;
+
+import static com.mongodb.ClusterFixture.isDataLakeTest;
+import static com.mongodb.ClusterFixture.isDiscoverableReplicaSet;
+import static com.mongodb.ClusterFixture.isServerlessTest;
+import static com.mongodb.ClusterFixture.isSharded;
+import static com.mongodb.ClusterFixture.serverVersionLessThan;
+import static com.mongodb.client.unified.UnifiedTestSkips.Modifier.IGNORE_EXTRA_EVENTS;
+import static com.mongodb.client.unified.UnifiedTestSkips.Modifier.SLEEP_AFTER_CURSOR_CLOSE;
+import static com.mongodb.client.unified.UnifiedTestSkips.Modifier.SLEEP_AFTER_CURSOR_OPEN;
+import static com.mongodb.client.unified.UnifiedTestSkips.Modifier.WAIT_FOR_BATCH_CURSOR_CREATION;
 import static org.junit.jupiter.api.Assumptions.assumeFalse;
 
 public final class UnifiedTestSkips {
+    public static void doSkips(final TestDef def) {
+
```

Comment:
The primary benefit is that one is able to more easily see what test-specific custom logic the driver has, without needing to check multiple files, many of which may not actually have their own custom logic.

It also incrementally moves us to having a single file for tests, rather than multiple files.